src/isPalindrome.js

```diff
@@ -19,7 +19,10 @@
  * @return {boolean}
  */
 function isPalindrome(str) {
-  // write code here
+  const regularExpression = /[\W_]/g;
```

Comment:
this name is too generic. name should explain what this regularExpression does. It extracts alphabetic charcters, so perhaps aplhabeticCharsRegExp